FlashLFQ/ChromatographicPeak.cs

```diff
@@ -6,17 +6,19 @@
 
 namespace FlashLFQ
 {
+    public enum PeakType { MSMS, MBR, Imputed }
```

Comment:
Break this out into another `.cs` file? This might make it easier to find, kind of like `Proteomics.Fragmentation.FragmentationTerminus` or something.

Reply:
This is an improvement over the previous boolean value, too 👍🏼